File: frontend/src/dtos/Autor/create/AutorCreateRequestDto.ts

export default class AutorCreateRequestDto
{
	nombre : string='';
	apellido : string='';
	cuit : number=0;
	biografia : string='';

	setNombre : (nombre : string) => AutorCreateRequestDto = (nombre: string) => {
		this.nombre = nombre;
		return this;
	}

	setApellido : (apellido : string) => AutorCreateRequestDto = (apellido: string) => {
		this.apellido = apellido;
		return this;
	}

	setCuit : (cuit : number) => AutorCreateRequestDto = (cuit: number) => {
		this.cuit = cuit;
		return this;
	}

	setBiografia : (biografia : string) => AutorCreateRequestDto = (biografia: string) => {
		this.biografia = biografia;
		return this;
	}
}

File: backend/src/main/java/com/example/template/dtos/AutorCreateRequestDTO.java
package com.example.template.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.util.*;
import com.example.template.enums.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AutorCreateRequestDTO {
private String Nombre;
private String Apellido;
private long Cuit;
private String Biografia;
}
